example/src/testing/Tests/webcryptoTests/sign_verify.ts

```diff
@@ -0,0 +1,167 @@
+import type { CryptoKey, CryptoKeyPair } from '../../../../../src/keys';
+import crypto from 'react-native-quick-crypto';
+import { describe, it } from '../../MochaRNAdapter';
+import { expect } from 'chai';
+
+// polyfill encoders
+// @ts-expect-error
+import { polyfillGlobal } from 'react-native/Libraries/Utilities/PolyfillFunctions';
+import RNFE from 'react-native-fast-encoder';
+polyfillGlobal('TextEncoder', () => RNFE);
```

Comment:
should this be part of a global setup/teardown? (I think the project has one already if im not mistaken)

Reply:
thanks... fix is going into #349